CartMoveController.py

```diff
@@ -0,0 +1,90 @@
+import RPi.GPIO as GPIO
+import time
+
+class CartMoveController(object):
+    def __init__(self, speed):
+        self.speed = speed
+        # Set the mode to GPIO.Board
+        GPIO.setmode(GPIO.BOARD)
+
+        # Output GPIO pins for the left side of the cart
+        GPIO.setup(32, GPIO.OUT) # A_IN_1
+        GPIO.setup(36, GPIO.OUT) # A_IN_2
+
+        # Output GPIO pins for the right side of the cart
+        GPIO.setup(38, GPIO.OUT) # B_IN_1
+        GPIO.setup(40, GPIO.OUT) # B_IN_2
+
+        # Output GPIO pins to control the speed of the wheels
+        GPIO.setup(29, GPIO.OUT) # PWM
+
+        # Set pin high
+        GPIO.output(29, GPIO.HIGH)
+
+        # Set frequency to 1kHz
+        self.pwm = GPIO.PWM(29, 1000)
+
+        # After relating speed to duty cycle
+        self.pwm.start(speed)
+        
+    def __del__(self):
+        self.pwm = 0
+        GPIO.cleanup()
+        
+    # Reverse Motion for t seconds
+    def reverse_motion(self, t):
```

Comment:
CartMoveController.py:35:4: C0103: Argument name "t" doesn't conform to snake_case naming style (invalid-name)

t is for the British!